tutorials/misc/plot_ecog.py

```diff
@@ -183,6 +157,87 @@ def animate(i, activity):
                                frames=show_power.shape[1],
                                interval=100, blit=True)
 
+###############################################################################
+# Now let's animate the raw time series data and add events related
+# to seizure onset that are marked in the annotations. This will be done
+# very similarly to the high gamma visualization
+
+# For this plot we will use a different angle (from left side)
+fig = plot_alignment(raw.info, subject='fsaverage', subjects_dir=subjects_dir,
+                     surfaces=['pial'], trans=None, coord_frame='head')
+mne.viz.set_3d_view(fig, -360, 90)
+
+xy, im = snapshot_brain_montage(fig, raw.info)
+
+# Convert from a dictionary to array to plot
+xy_pts = np.vstack([xy[ch] for ch in raw.info['ch_names']])
+
+# colormap to view spectral power - RdBu_r works well for directional
+# data such as voltage time series, where positive and negative values
+# matter and there is a true 0 point.
+cmap = 'RdBu_r'
+
+raw_notched = raw.copy().notch_filter([60, 120])
+ts_data = raw_notched.get_data()
+# Create a dictionary of seizure-related events from raw.annotations
+# These will be used to update the animation and show the most relevant
+# annotation
+events = dict()
+for t in np.arange(len(raw.annotations)):
```

Comment:
This looks a lot like [mne.events_from_annotations](https://mne.tools/stable/generated/mne.events_from_annotations.html)